Makefile

```diff
@@ -143,12 +143,10 @@ test-npm: $(NODE_EXE)
 	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
 	     npm_config_prefix="$(shell pwd)/npm-prefix" \
 	     npm_config_tmp="$(shell pwd)/npm-tmp" \
-	     ../../$(NODE_EXE) cli.js install --ignore-scripts
-	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
-	     npm_config_prefix="$(shell pwd)/npm-prefix" \
-	     npm_config_tmp="$(shell pwd)/npm-tmp" \
+	     ../../$(NODE_EXE) cli.js install --ignore-scripts && \
```

Comment:
The bundled npm doesn't include the `devDependencies` (and npm doesn't bundle the `devDependencies` in the version on the registry, either). They need to be installed before the tests can be run (obvs), but another way to do it would be to copy `deps/npm` elsewhere and then do the `npm install` on that, nuking it after the tests have run. I'm increasingly of the opinion that the best way to do this is with a shell script called from the `Makefile`, as this is getting pretty complicated (and hard to follow) for a `Makefile` stanza.